### PR TITLE
@swalkinshaw's latest deploy updates from roots.io

### PR DESCRIPTION
**Pull request by retlehs**

### CODE REVIEW

File: roles/deploy/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  git: "repo={{ project_git_repo }} dest={{ project_source_path }} version={{ project_version }}"
+
+- name: write unfinished file
+  file: "path={{ project_source_path }}/{{ deploy_helper.unfinished_filename }} state=touch"
+
+- name: Copy files to new build dir
+  command: "cp -pr {{ project_source_path }} {{ deploy_helper.new_release_path }}"
+
+- name: Move site into root
+  shell: "mv {{ deploy_helper.new_release_path }}/roots.io/* {{ deploy_helper.new_release_path }}"
```
Comment (retlehs) [inline roles/deploy/tasks/main.yml]: oops

**Comment by fullyint** [comment]
I can't find any evidence of this merge on the `deploys` branch. Any ideas of what I'm missing?

**Comment by retlehs** [comment]
haha. @swalkinshaw accidentally force pushed the deloys branch the other day and overwrote this

i'll let him figure it out :rage1:

**Comment by fullyint** [comment]
😊 Thanks! Makes sense now.

**Pull request merged**